### ISSUE TITLE
[templates] templateinstance readiness test [It] should report ready soon after all annotated objects are ready

### ISSUE THREAD

**Issue by bparees** [openshift/origin#15901]
```
[templates] templateinstance readiness test
/go/src/github.com/openshift/origin/test/extended/templates/templateinstance_readiness.go:193
  should report ready soon after all annotated objects are ready [It]
  /go/src/github.com/openshift/origin/test/extended/templates/templateinstance_readiness.go:142

  Expected error:
      <*errors.errorString | 0xc42026f540>: {
          s: "timed out waiting for the condition",
      }
      timed out waiting for the condition
  not to have occurred

  /go/src/github.com/openshift/origin/test/extended/templates/templateinstance_readiness.go:125
```

https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_templates/69

Guessing this is a flake since it runs as part of the merge queue, so it shouldn't have been explicitly broken by anything.

**Comment by jim-minter** [comment]
AFAICS, cakephp-mysql-example-1-hook-pre terminated successfully, but nothing happened next.  Master logs are missing so can't diagnose further.

**Comment by bparees** [comment]
any debug we should add to the test to better dump the various pod states/information when this fails in the future?

**Comment by jim-minter** [comment]
On https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_extended_templates/754/, the build failed due to being unable to push to the registry. Master logs missing. https://github.com/openshift/origin/issues/15903.

**Comment by jim-minter** [comment]
> any debug we should add to the test to better dump the various pod states/information when this fails in the future?

possibly, but I'm going to attack getting the master logs first.

**Comment by bparees** [comment]
dunno that there's anything more useful in this one, but it looks like the same failure occurred here:
https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_templates/68/

**Comment by jim-minter** [comment]
On https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_templates/68/ the build failed for some reason.  https://github.com/openshift/aos-cd-jobs/pull/581 re-added the master log collection.  I'll add dumping of the build & deploy states to this job as well.

**Comment by jim-minter** [comment]
I believe these issues were fixed by #15966
